editor/editor_feature_profile.h

```diff
@@ -147,7 +147,8 @@ class EditorFeatureProfileManager : public AcceptDialog {
 	String current_profile;
 	void _update_profile_list(const String &p_select_profile = String());
 	void _update_selected_profile();
-	void _fill_classes_from(TreeItem *p_parent, const String &p_class, const String &p_selected);
+	void _update_profile_tree_from(TreeItem *p_edited);
+	void _fill_classes_from(TreeItem *p_parent, const String &p_class, const String &p_selected, const int p_class_insert_index = -1);
```

Comment:
```suggestion
	void _fill_classes_from(TreeItem *p_parent, const String &p_class, const String &p_selected, int p_class_insert_index = -1);
```
No `const` for simple types